projects/valtimo/components/src/lib/components/multiselect-dropdown/multiselect-dropdown.component.scss

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Why include this file if it is empty? At least remove empty class

Reply:
I found multiple empty scss files so I thought that was the norm. I'll remove the empty class

Reply:
Empty scss file is fine I agree, just remove the empty class